src/Resources/js/modules/custom-functions.js

```diff
@@ -40,7 +40,10 @@ Mealz.prototype.copyToClipboard = function() {
             } else {
                 guestMenuLinkInput.trigger('select');
             }
-            navigator.clipboard.writeText(result);
+            // Clipboard copy only works in secure context, redirect http -> https
+            if(window.isSecureContext) {
+                navigator.clipboard.writeText(result);
+            }
```

Comment:
We can improve it a bit more by adding "console.warn('Clipboard access from insecure (HTTP) context is prohibited. Reload page with HTTPS and try again.')" in else block. The comment above can then be removed.

Reply:
+1